pandas-stubs/core/tools/timedeltas.pyi

```diff
@@ -31,8 +31,7 @@ def to_timedelta(
 @overload
 def to_timedelta(
     arg: (
-        Sequence[float | timedelta]
-        | list[str | float | timedelta]
+        Sequence[str | float | timedelta]
```

Comment:
The CI is failing because a plain string will match `Sequence[str]` .  So you should change this to 
```suggestion
        SequenceNotStr | Sequence[float | timedelta]
```
You can import `SequenceNotStr` from `_typing`

Also, you should set up your environment and test locally.

Reply:
Great that you are confirming my initial solution, `'1 day'` and `['1 day']` are both `Sequence`.
I forgot to rerun locally when I pushed the feedback last time, should be good now.
Thanks for all the insights on my first PR here!

Reply:
I had to force the typehint in the test, let me know if this is in the spirit.